core/domain/user_jobs_one_off_test.py

```diff
@@ -1354,6 +1354,31 @@ def test_that_last_edited_and_created_time_are_not_updated(self):
         self.assertIsNone(owner_settings.last_edited_an_exploration)
 
 
+class FillExplorationIdsInUserSubscriptionsModelOneOffJobTests(test_utils.GenericTestBase): # pylint: disable=line-too-long
+
+    def _run_one_off_job(self):
+        """Runs the one-off MapReduce job."""
+        job_id = (
+            user_jobs_one_off.FillExplorationIdsInUserSubscriptionsModelOneOffJob.create_new()) # pylint: disable=line-too-long
+        user_jobs_one_off.FillExplorationIdsInUserSubscriptionsModelOneOffJob.enqueue(job_id) # pylint: disable=line-too-long
+        self.assertEqual(
+            self.count_jobs_in_mapreduce_taskqueue(
+                taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
+        self.process_and_flush_pending_mapreduce_tasks()
+        stringified_output = (
+            user_jobs_one_off.FillExplorationIdsInUserSubscriptionsModelOneOffJob.get_output(job_id)) # pylint: disable=line-too-long
+        eval_output = [ast.literal_eval(stringified_item) for
+                       stringified_item in stringified_output]
+        return eval_output
+
+    def test_that_exploration_ids_and_activity_ids_are_equal(self):
+        # Generate activity_ids.
+        user_models.UserSubscriptionsModel(
+            activity_ids=['exp_1', 'exp_2', 'exp_3']).put()
+        exploration_ids = self._run_one_off_job()
+        self.assertEqual([[u'SUCCESS', 1]], exploration_ids)
```

Comment:
```suggestion
        subscription_model = user_models.UserSubscriptionsModel(
            activity_ids=['exp_1', 'exp_2', 'exp_3']
        )
        subscription_model.update_timestamps()
        subscription_model.put()
        output = self._run_one_off_job()
        self.assertEqual([[u'SUCCESS', 1]], output)
        self.assertEqual(
            subscription_model.activity_ids, subscription_model.exploration_ids)
```

Reply:
Also, add a similar test but when `activity_ids` is `[]` and for multiple `UserSubscriptionsModel`

Reply:
@vojtechjelinek, how will `subscription_model` have the updated `exploration_ids`? I tried this approach but the `exploration_ids` of the `subscription_model` is still empty, whereas `model_instance` in `user_jobs_one_off.py `updates correctly.

Reply:
Right, I have edited the code above, now it should work.

Reply:
Done.